consensus/hotstuff/votecollector/combined_aggregator.go

```diff
@@ -0,0 +1,54 @@
+package votecollector
+
+import (
+	"go.uber.org/atomic"
+
+	"github.com/onflow/flow-go/crypto"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// combinedAggregator aggregates signatures from different signers for the same message.
+// The instance must be initialized with the message, the required weight to be
+// sufficient for aggregation, as well as the weight for each signer.
+type combinedAggregator struct {
```

Comment:
What interface we are implementing here? I thought it should be `SignatureAggregator` but it has too many functions for it.

Reply:
Seems obsolete